internal/cli/get/projects.go

```diff
@@ -0,0 +1,70 @@
+package get
+
+import (
+	goerrors "errors"
+
+	"connectrpc.com/connect"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"golang.org/x/exp/slices"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	typesv1alpha1 "github.com/akuity/kargo/internal/api/types/v1alpha1"
+	"github.com/akuity/kargo/internal/cli/client"
+	"github.com/akuity/kargo/internal/cli/option"
+	v1alpha1 "github.com/akuity/kargo/pkg/api/service/v1alpha1"
+)
+
+func newProjectsCommand(opt *option.Option) *cobra.Command {
```

Comment:
This confused me at first. The name appears as if it could be responsible for the `kargo new project` command. If we named it `newGetProjectsCommand` it might avoid that confusion.

Same applies for the other new subcommands that are introduced.